lib/mws/fulfillment_inventory/client.rb

```diff
@@ -9,7 +9,8 @@ module FulfillmentInventory
     # selling your inventory on Amazon's retail web site or through other retail
     # channels.
     class Client < ::Peddler::Client
-      path '/FulfillmentInventory/2010-10-01'
+      version '2010-10-01'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.